packages/stargate/src/stargateclient.ts

```diff
@@ -62,6 +62,7 @@ export interface BroadcastTxSuccess {
   readonly transactionHash: string;
   readonly rawLog?: string;
   readonly data?: readonly MsgData[];
+  readonly gasUsed: number;
```

Comment:
Hmm, BroadcastTxSuccess does not mean it was executed. If we assume this type is only used for `broadcastTxCommit`, we should make it specific to that case.